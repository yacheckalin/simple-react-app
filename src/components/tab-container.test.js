import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import TabsContainer from "./tab-container";

describe("<TabsContainer />", () => {
  test("displays and shows properly", () => {
    const wrapper = shallow(<TabsContainer />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
