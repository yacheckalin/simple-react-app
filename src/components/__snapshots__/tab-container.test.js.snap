// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabsContainer /> displays and shows properly 1`] = `
<Suspense
  fallback="loading ..."
>
  <styled.div>
    <div
      className="stretcher"
    >
      <div
        className="tab-wrapper"
      >
        <styled.a
          active={1}
          onClick={[Function]}
        >
          tabs.currency
        </styled.a>
        <styled.a
          active={0}
          onClick={[Function]}
        >
          tabs.calculator
        </styled.a>
        <styled.a
          active={0}
          onClick={[Function]}
        >
          tabs.history
        </styled.a>
      </div>
      <CurrencyContextProvider>
        <Memo() />
      </CurrencyContextProvider>
    </div>
  </styled.div>
</Suspense>
`;
